src/Service/Backend.ts

```diff
@@ -165,6 +165,34 @@ export class Backend {
         return response
     }
 
+    /**
+     * Prepare a Smart Wallet transaction by augmenting it with mandatory witnesses.
+     * @async
+     * @param {PrepareTxParameters} params - Request parameters for the prepare-tx endpoint
+     * @returns {PrepareTxResponse}
+     */
+    async prepareTx(params: PrepareTxParameters): Promise<PrepareTxResponse> {
+        const payload = serialize({
```

Comment:
won't this work? I see that field names are the same
```suggestion
        const payload = serialize(params)
```

Reply:
Agree.